src/triggers/issue.ts

```diff
@@ -63,7 +74,8 @@ const buildIssueList = (orderBy: "createdAt" | "updatedAt") => async (z: ZObject
       ) {
         team(id: $teamId) {
           issues(
-            first: 25
+            first: 10
```

Comment:
This endpoint is polled, most of the time there won't be many issues created since the last poll so it makes sense to keep the pagination limit and complexity low.